salsa/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "salsa"
+version = "0.1.0"
+authors = ["Eric McCorkle <eric@metricspace.net>"]
+license = "BSD"
```

Comment:
Looks like this needs updated per @newpavlov. I saw the `LICENSE-APACHE` and `LICENSE-MIT` below, so I'm guessing this is copypasta.